src/src/WebServer/ESPEasy_WebServer.cpp

```diff
@@ -743,7 +743,7 @@ bool isLoggedIn(bool mustProvideLogin)
 
 String getControllerSymbol(uint8_t index)
 {
-  String ret = F("<p style='font-size:20px; background: #00000000;'>&#");
+  String ret = F("<p style='font-size:20px; background: #00000000;line-height:21px;'>&#");
```

Comment:
a suggestion:
<img width="360" alt="Bildschirmfoto 2022-09-04 um 11 20 14" src="https://user-images.githubusercontent.com/33860956/188306550-080ff736-55e8-4055-b887-3f3d3ba23601.png">
```
<td>Send to: <span style="font-size:20px; background: #00000000;">❶</span> 
    Domoticz HTTP 
</td>
```
..and using <span> doesn’t break the controller number anywhere else as far as i can see

Reply:
It was sitting at the bottom of the line, not nicely vertical-centered with the checkbox, using Chrome on Windows. The misalignment had to do with line-height, so I reduced that.
Will try with `span` instead of `p` group, to see if that improves things.

Reply:
Fixed, restored layout close to previous layout, see below for screenshot.